packages/fast-components-styles-msft/src/utilities/format.spec.ts

```diff
@@ -0,0 +1,51 @@
+import designSystemDefaults, { DesignSystem } from "../design-system";
+import { format } from "./format";
+
+function formatter(designSystem: DesignSystem): string {
+    return "%formatted%";
+}
+
+describe("format", (): void => {
+    test("should return the input string if no formatters are passed", (): void => {
+        const input: string = "foobar{0}";
+        expect(format(input)(designSystemDefaults)).toBe(input);
+    });
+    test("should format '{0}' when a formatter is passed as the second argument", (): void => {
+        expect(format("{0}", formatter)(designSystemDefaults)).toBe(
+            formatter(designSystemDefaults)
+        );
+    });
+    test("should format multiple '{0}'", (): void => {
+        expect(format("{0}{0}", formatter)(designSystemDefaults)).toBe(
+            formatter(designSystemDefaults) + formatter(designSystemDefaults)
+        );
+    });
+    test("should not format a variable when not supplied a formatter", (): void => {
+        expect(format("{0}{1}", formatter)(designSystemDefaults)).toBe(
+            formatter(designSystemDefaults) + "{1}"
+        );
+    });
+    test("should format multiple variables when multiple formatters are passed", (): void => {
+        expect(format("{0}{1}", formatter, formatter)(designSystemDefaults)).toBe(
```

Comment:
I would suggest using a different formatter for the second formatter, just to make sure there isn't an issue where the wrong formatter gets used